net-analyzer/goaccess/goaccess-1.6.2.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools systemd tmpfiles
+
+DESCRIPTION="A real-time web log analyzer and interactive viewer in a terminal"
+HOMEPAGE="https://goaccess.io"
+SRC_URI="https://tar.goaccess.io/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE="debug geoip geoipv2 getline ssl unicode"
+REQUIRED_USE="geoipv2? ( geoip )"
+
+RDEPEND="acct-user/goaccess
+	sys-libs/ncurses:=[unicode(+)?]
```

Comment:
You're missing the dependency on group.

Reply:
@juippis I see that you added it. Do we really need it?
I have ACCT_USER_GROUPS sets and there is a call of acct-user_add_deps, which should do [this](https://github.com/gentoo/gentoo/blob/master/eclass/acct-user.eclass#L164).

Or there is some policy that requires include group in ebuild RDEPEND explicitly?

Reply:
We generally don't rely on indirect deps. If something's needed by your package, and not directly inherited by an eclass, I'd say you're better listing it in the ebuild. IMHO it makes things consistent and easier to grasp in the ebuild too, and is future-proof too.

Reply:
Got it. Thanks.